xml/FrameworksIndex/net-7.0.xml

```diff
@@ -47593,8 +47593,10 @@
       <Member Id="M:System.Text.Json.Serialization.Metadata.JsonTypeInfo.CreateJsonPropertyInfo(System.Type,System.String)" />
       <Member Id="M:System.Text.Json.Serialization.Metadata.JsonTypeInfo.CreateJsonTypeInfo(System.Type,System.Text.Json.JsonSerializerOptions)" />
       <Member Id="M:System.Text.Json.Serialization.Metadata.JsonTypeInfo.CreateJsonTypeInfo``1(System.Text.Json.JsonSerializerOptions)" />
+      <Member Id="M:System.Text.Json.Serialization.Metadata.JsonTypeInfo.MakeReadOnly" />
```

Comment:
@carlossanlop It looks like there are only 3 new APIs since RC1.

The rest of the changes are either doc formatting (newlines) or new attribute annotations (MaybeNullWhen and EditorBrowsableState.Never).

Reply:
Yeah, after Preview7 we were not introducing new APIs except in a few last-minute urgent cases.

There were some new APIs introduced in GA by the way, so I'll have to send you a new set of ref assemblies next month. Very special and unusual case.

Reply:
Do you mean they were introduced after the snap for GA? I'm confused.